lib/filter-options.tsx

```
@@ -0,0 +1,174 @@
+  { label: "Medium", value: "medium", searchParam: "medium" },
+  { label: "Low", value: "low", searchParam: "low" }
+]
+
+// Default error categories to be selected (ticked) by default
+export const defaultSelectedErrorCategories = ["unknown_error"];
+
+/**
+ * Generate dynamic error category options based on actual error data
+ * This replaces static categories with real data from the system
+ */
+export function generateDynamicErrorCategoryOptions(errorBots: any[]): Option[] {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)